### PR TITLE
doc: fix standalone stop cannot start issue

### PR DESCRIPTION
**Pull request by pingliu**

### CODE REVIEW

File: scripts/standalone_embed.sh [hunk 1]
```diff
@@ -41,6 +41,23 @@ EOF
...
+    if [ $res -eq 1 ]
+    then
+        echo "Milvus is running."
+        exit 0
+    fi
+
+    res=`sudo docker ps -a|grep milvus-standalone|wc -l`
+    if [ $res -eq 1 ]
+    then
+        sudo docker start milvus-standalone 1> /dev/null
```
Comment (LoveEachDay) [inline scripts/standalone_embed.sh:57]: We'd better extract the wait_for_milvus_running as a utility helper function. Add it to `docker start` too.

**Review by LoveEachDay — APPROVED** [review]
/lgtm

File: scripts/standalone_embed.sh (hunk 1 above)
Comment (pingliu) [reply scripts/standalone_embed.sh:71]: changed

**Pull request merged**